tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -521,8 +526,10 @@ JSONObject getPermissionsJson() throws JSONException {
                     .put("userCanUseVettingParticipation", userCanUseVettingParticipation(this))
                     .put("userIsAdmin", userIsAdmin(this))
                     .put("userIsManager", getLevel().canManageSomeUsers())
-                    .put("userIsTC", userIsTC(this))
-                    .put("userIsVetter", userIsVetter(this) && !userIsTC(this))
+                    .put("userIsTC", userIsTCOrStronger(this))
+                    // Caution: userIsVetter really means user is Vetter or Manager, but not TC or
+                    // Admin. It should be renamed to avoid confusion.
```

Comment:
```suggestion
```

Reply:
I think the comment should stay until the json item is renamed or otherwise refactored